src/lib.rs

```diff
@@ -79,21 +79,21 @@
 //! embedded graphics code, or produce examples and interactive demos to show of embedded graphics
 //! features.
 //!
-//! ![It can display all sorts of embedded-graphics test code.](https://raw.githubusercontent.com/jamwaffles/embedded-graphics/master/assets/simulator-demo.png)
+//! ![It can display all sorts of embedded-graphics test code.](https://raw.githubusercontent.com/embedded-graphics/embedded-graphics/master/assets/simulator-demo.png)
```

Comment:
I'm not sure I mentioned this in the `simulator` PR review: Should we move the image to the simulator repo?

Reply:
Yes, I think so, along with #402

Reply:
I added a note to #402 to track.